mlem/cli/main.py

```diff
@@ -481,20 +488,32 @@ def inner(*iargs, **ikwargs):
     "--target-project",
     "--tp",
     help="Project to save target to",
+    metavar=PATH_METAVAR,
     show_default="none",  # type: ignore
 )
 option_json = Option(False, "--json", help="Output as json")
 option_data_project = Option(
     None,
     "--data-project",
     "--dr",
+    metavar=PATH_METAVAR,
     help="Project with data",
 )
 option_data_rev = Option(
     None,
     "--data-rev",
     help="Revision of data",
 )
+option_model = Option(
+    ...,
+    "-m",
+    "--model",
+    help="Path to MLEM model object",
+    metavar=PATH_METAVAR,
+)
+option_data = Option(
+    ..., "-d", "--data", help="Path to MLEM data object", metavar=PATH_METAVAR
```

Comment:
Clearer 👍🏼 though maybe "model/data MLEM object" would be a better phrasing (keep "MLEM object" together). No strong opinion

Reply:
@aguschin wdyt?

Reply:
IMO `MLEM model object` is simpler than `model MLEM object`.

Reply:
I think it should be "Path to MLEM model" (ppl don't care about "object", it doesn't give additional information).

Reply:
Is it .mlem path or path to an actual directory / file that MLEM creates? MLEM model object ideally should be just "Path to model"

Reply:
I think it's the path to the .mlem file which is why I suggested somewhere else to add `(.mlem file)` just to be specific.

Reply:
No, it's a path to an actual model. E.g. if you have `model.pkl` and `model.pkl.mlem` nearby, the argument should be `model.pkl` (although it should work if one supplies `model.pkl.mlem` as well).

The thing is, if there is no `model.pkl.mlem`, MLEM won't understand what it is. So it should be MLEM object (== `model.pkl.mlem` should exist) for this to work, not just a binary saved with pickle/joblib or csv file saved with pandas.

Reply:
What @aguschin said is correct, I just see it the other way around - you should specify path to `.mlem` file, but omitting `.mlem` suffix also works. But if somehow you have `aaa.mlem` that uses `bbb.pkl` file, you can't specify `bbb.pkl` - you can use `aaa.mlem` or `aaa`. But generally `bbb.pkl` == `aaa`, unless you tamper with mlem files manually.
As for what @shcheklein said: I added `MLEM Object` to imply that it only works with models saved by MLEM. We can omit it, but I think this will make it less clear. We can also do `Path to MLEM model` probably

Reply:
`Path to MLEM model` and `Path to MLEM dataset` sounds good to me.

Reply:
I see this is merged but just FYI this is similar to `dvc push` (and other commands) which `targets` can be either tracked files/dirs or .dvc files. See e.g. https://dvc.org/doc/command-reference/push for ideas on good phrasing around this.